tests/models/nn/transformers/test_utils.py

```diff
@@ -0,0 +1,75 @@
+#  Copyright 2022-2025 MTS (Mobile Telesystems)
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# pylint: disable=attribute-defined-outside-init
+
+import typing as tp
+
+import pandas as pd
+import pytest
+
+from rectools import Columns
+from rectools.dataset import Interactions
+from rectools.models.nn.transformers.utils import leave_one_out_mask
+
+
+class TestUtils:
```

Comment:
```suggestion
class TestLeaveOneOutMask
```
We should name tests depending on the names of tested classes/functions.